tests/test_exposition.py

```diff
@@ -373,7 +362,12 @@ def _expect_metric_exception(registry, expected_error):
 def test_basic_metric_families(registry, MetricFamily, value, error):
     metric_family = MetricFamily(MetricFamily.__name__, 'help')
     registry.register(Collector(metric_family, value))
-    _expect_metric_exception(registry, error)
+    try:
+        generate_latest(registry)
+    except error as exception: 
+        # Got a valid error as expected, return quietly
```

Comment:
Can we check the type of exception is correct?

Reply:
Unless I'm misunderstanding, we are: `error` is a type which pytest is parametrizing to be `TypeError` or `ValueError`

Reply:
Right you are